lib/src/offline_region.dart

```diff
@@ -1,5 +1,38 @@
 part of mapbox_gl;
 
+/// Description of region to be downloaded. Identifier will be generated when
+/// the download is initiated.
+class OfflineRegionDefinition {
```

Comment:
adding a fromOfflineRegion() might be pretty useful - if we get the list of the currently downloaded regions, it would probably be desiderable to be able to recreate the region definition. 

Another option might be to make OfflineRegionDefinition the baseclass and have OfflineRegion extend it with the id field. With this one could simply cast a OfflineRegion to a OfflineRegionDefinition

Reply:
The base class option works nicely and maintains API compatibility, I'll push a change with that.